src/components/Toolbar/Index.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+    StyleSheet,
+    ToolbarAndroid,
+    ToolbarAndroidAction
+} from 'react-native';
+import * as React from 'react';
+import icons, {
+    images
+} from '../../assets/index';
```

Comment:
I believe we can just use `'../../assets` because the name of the file is `index`.

Reply:
you're right